packages/evm/src/precompiles/14-kzg-point-evaluation.ts

```diff
@@ -37,14 +49,6 @@ export async function precompile14(opts: PrecompileInput): Promise<ExecResult> {
   const commitment = opts.data.subarray(96, 144)
   const kzgProof = opts.data.subarray(144, 192)
 
```

Comment:
Why did you remove this check?  This is part of the EIP.

Also, if we do agree to remove this, we should also remove the associated EVM error for inputs greater than the moduls.

Reply:
It is part of the EIP, but this check is actually done inside the C-KZG. If I keep it, then we have 26 failing tests. If I remove, it passes. (So our pre-input check which we have is wrong, it likely takes the wrong inputs to check against). I will remove the error message, indeed.

Reply:
![image](https://github.com/ethereumjs/ethereumjs-monorepo/assets/29359032/ff93eb83-1d99-4fb2-b9e4-8c9eb18601de)

I asked this sometime ago on R&D

Reply:
ahh yes, this was anyway wrong sice y and z were coming in little endian and here our bytesToBigInt is big endian, and verify from kzg has these checks as mentioned by Jochem so we can remove them and let underlying libs handle the correctness